### DISCUSSION TITLE
Dynare 6.0 Release in Homebrew

### DISCUSSION THREAD

**Discussion by SantiagoD999** [Homebrew/discussions#5118] [Everyday usage]
### Output of `brew config`

```shell
-
```

### Output of `brew doctor`

```shell
-
```

### Description of issue

Hello to all,

I am using the Homebrew version of Dynare, the latest version of the software is the 6.0, but in Homebrew the latest version available is 5.5. Does anyone know when the latest version will be available?

Thank you very much,

**Comment by gromgit** [comment]
> Does anyone know when the latest version will be available?

When someone who's familiar with Dynare compilation helps fix the build process: https://github.com/Homebrew/homebrew-core/pull/161676
